src/main/webapp/data-visualization.js

```diff
@@ -156,28 +156,26 @@ function createDataArray(censusDataArray, isCountyQuery) {
 // percentToTotal takes in the total number of people in a category
 // and the percentage and returns the total
 function percentToTotal(totalNumber, percentage) {
-  return (totalNumber/100) * percentage;
+  return Math.round((totalNumber/100) * percentage);
 }
 
-
 // checkPercentage() Takes in the header of a census query and returns
 // whether or not the total needs to be calculated using the
 // percentToTotal() function
-function checkPercentage(headerColumn) {
-  // percentageQueries is a list of queries that return percents and not raw
-  // data. It is hard coded for now.
-  const percentageQueries = ['S0201_157E', 'S0201_126E'];
-  let i;
-  // Iterate through all the data variables in the header.
-  // Eg: ["NAME","S0201_119E","S0201_126E","state"]
-  // We need to return true
-  for (i = 1; i < headerColumn.length - 1; i++) {
-    // if the current data is in the percentage array
-    if (percentageQueries.indexOf(headerColumn[i]) > -1) {
-      return true;
-    }
+function checkPercentage(headerColumn, isCountyQuery) {
+  // percentageQueries is a list of queries that return percents and not raw data.
```

Comment:
percentageQueries is no longer a variable in this function so it might be helpful to update the comment with only explanations for why percentage queries have length 5 instead of 4

Reply:
Nice catch, thanks.